content/it/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Immagini
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+L'immagine di un container rappresenta dati binari che incapsulano un'applicazione e 
+tutte le sue dipendenze software. Le immagini sono costituite da pacchetti software 
+eseguibili che possono essere avviati in modalità standalone e su cui si possono fare 
+ipotesi ben precise circa l'ambiente in cui vengono eseguiti.
+
+Tipicamente viene creata un'immagine di un'applicazione ed effettuato il _push_ 
+su un registro prima di poterne fare riferimento esplicito in un 
+{{< glossary_tooltip text="Pod" term_id="pod" >}}
+
+Questa pagina va a delineare nello specifico il concetto di immagine di un container.
+
+<!-- body -->
+
+## I nomi delle immagini
+
+Alle immagini dei container vengono normalmente attribuiti nomi come `pausa`, `esempio/ilmiocontainer`, o `kube-apiserver`.
```

Comment:
```suggestion
Alle immagini dei container vengono normalmente attribuiti nomi come `pause`, `example/mycontainer`, o `kube-apiserver`.
```
No need to translate those examples, the are quite clear also in English

Reply:
`pause` is a kind of hard-coded name, BTW.

Reply:
in this case, I would revert to the initial commit